src/libs/ReportUtils.ts

```diff
@@ -2257,14 +2257,14 @@ function isPayer(session: OnyxEntry<Session>, iouReport: OnyxEntry<Report>, only
     const isManager = iouReport?.managerID === session?.accountID;
     if (isPaidGroupPolicy(iouReport)) {
         if (policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES) {
-            // If we get here without a reimburser only show the pay button if we are the manager.
+            // If we get here without a reimburser only show the pay button if we are the admin.
             if (!policy?.achAccount?.reimburser) {
-                return isManager;
+                return isAdmin && isManager;
```

Comment:
Puneet, do we have any context on OldDot for this specific condition?
If there's no `reimburser`, is it necessary to check both `isAdmin && isManager`, or is just the `isAdmin` check sufficient here?